rust/chains/hyperlane-fuel/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "hyperlane-fuel"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+fuels = "0.31"
+hyperlane-core = { path = "../../hyperlane-core" }
+async-trait = { version = "0.1", default-features = false }
+thiserror = "1.0"
+
+# TODO: use the version in hyperlane-core once we merge https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/1387
```

Comment:
This was merged right?

Reply:
Yep, but not when I wrote that, good catch